src/Microsoft.OpenApi/Services/OpenApiWalker.cs

```diff
@@ -391,11 +394,17 @@ internal void Walk(OpenApiContact contact)
         /// </summary>
         internal void Walk(OpenApiCallback callback, bool isComponent = false)
         {
-            if (callback == null || ProcessAsReference(callback, isComponent))
+            if (callback == null)
```

Comment:
Let's chat about this change. I'm very concerned about changing OpenApiWalker behavior.  I'm not saying this is wrong, I just need to have it explained slowly to me :-)